Configurations/15-ios.conf

```diff
@@ -33,6 +33,12 @@ my %targets = (
     "iossimulator-xcrun" => {
         inherit_from     => [ "ios-common" ],
         CC               => "xcrun -sdk iphonesimulator cc",
+        cflags           => add("-arch x86_64 -mios-simulator-version-min=7.0.0"),
+    },
+    "iossimulator-arm64-xcrun" => {
+        inherit_from     => [ "ios-common" ],
+        CC               => "xcrun -sdk iphonesimulator cc",
+        cflags           => add("-arch arm64 -mios-simulator-version-min=7.0.0"),
```

Comment:
Wouldn't you want to pass the other stuff from the actual iOS targets? Like -fno-common and the ASM stuff. Same goes for the x86_64 target, looking at what the macOS targets do.

Reply:
> Like -fno-common
Beside the iOS targets `-fno-common` is only used in "darwin-ppc" and "darwin-ppc-cc". I wonder if it is important?

> ASM stuff
Look like I can reuse `asm_arch` and `perlasm_scheme` from ios64-xcrun for iossimulator-arm64-xcrun, but I haven't been able to find the right setting for the x86_64 simulator. And I haven't tested if the build output. Maybe you can assist here?

Reply:
The right settings would probably be similar to x86_64 Darwin.

Reply:
I had to add `bn_ops           => "SIXTY_FOUR_BIT_LONG",`. Looks like it is working.